Checkpoint/checkpoint/ingest/parser.yml

```
@@ -169,6 +169,10 @@ stages:
           url.original: "{{checkpoint.message.request}}"
           user_agent.original: "{{checkpoint.message.user_agent}}"
 
+      - set:
+          action.tcp.flag: "{{checkpoint.message.cs4}}"
+        filter: "{{checkpoint.message.cs4Label == 'TCP Flags'}}"
```

Can be simplified with (as CEF stage also resolved the label associated to a field):
```suggestion
      - set:
          action.tcp.flag: "{{checkpoint.message.get('TCP Flags')}}"
```